src/utils/serve-functions.js

```diff
@@ -153,16 +155,12 @@ const buildClientContext = function (headers) {
   }
 }
 
-const logBeforeAction = ({ omitLog, path, action }) => {
```

Comment:
The `omitLog` property was introduced to avoid duplicate logging messages when using the ZISI function builder, but it's no longer necessary since the ZISI function builder now bypasses the base file watchers and thus removing duplicate events.